include/WLGDEventAction.hh

```diff
@@ -47,6 +45,27 @@ private:
   G4THitsMap<G4double>*      GetHitsCollection(G4int hcID, const G4Event* event) const;
   G4THitsMap<G4ThreeVector>* GetVecHitsCollection(G4int hcID, const G4Event* event) const;
 
+  std::vector<int> FilterTrajectories(int item, std::vector<G4int>& tid, std::vector<G4int>& pid)
```

Comment:
I think a brief outline of what this function does, including the inputs/outputs, would be worthwhile. For future-proofing, the doxygen style can be used:

```c++
//! Brief description
/*!
 * More detail if needed
 *
 * \param[in] item Description of parameter
 * \param[in] tid Description ...
 * \param[in] pid Description ... 
 */
std::vector<int> FilterTrajectories(int item, std::vector<G4int>& tid, std::vector<G4int>& pid)
```

I *think* the `tid` and `pid` parameters can be `const`

Reply:
Latest commit adds doxygen description of filter function.